tests/api/entities/resolver/resolve_and_update.test.js

```diff
@@ -182,6 +182,69 @@ describe('entities:resolver:update-resolved', () => {
     batchId.should.above(startTime)
     batchId.should.below(Date.now())
   })
+
+  it('should not update if entry date is as precise as entity date', async () => {
+    const year = '2020'
+    const { uri, isbn } = await createEditionWithIsbn({ publicationDate: year })
+    const entry = {
+      edition: {
+        isbn,
+        claims: { 'wdt:P577': year }
+      }
+    }
+    const { entities: preResolvedEntities } = await getByUris(uri)
+    const preResolvedEntityVersion = Object.values(preResolvedEntities)[0].version
+
+    await resolveAndUpdate(entry)
+    await wait(10)
+    const { entities: postResolvedEntities } = await getByUris(uri)
+    const postResolvedEntityVersion = Object.values(postResolvedEntities)[0].version
+    postResolvedEntityVersion.should.equal(preResolvedEntityVersion)
+  })
+
+  it('should update if entry date is more precise than entity date', async () => {
+    const entryDate = '2020-01-01'
+    const entityDate = '2020'
+    const { uri, isbn } = await createEditionWithIsbn({ publicationDate: entityDate })
+    const entry = {
+      edition: {
+        isbn,
+        claims: { 'wdt:P577': entryDate }
+      }
+    }
+    const { entities: preResolvedEntities } = await getByUris(uri)
+    const preResolvedEntityVersion = Object.values(preResolvedEntities)[0].version
+
+    await resolveAndUpdate(entry)
+    await wait(10)
+    const { entities: postResolvedEntities } = await getByUris(uri)
+    const postResolvedEntityVersion = Object.values(postResolvedEntities)[0].version
+    postResolvedEntityVersion.should.equal(preResolvedEntityVersion + 1)
```

Comment:
would be nice to also assert that the resulting date is the one expected:
```suggestion
    const postResolvedEntity = postResolvedEntities[uri]
    postResolvedEntity.version.should.equal(preResolvedEntityVersion + 1)
    postResolvedEntity.claims['wdt:P577'].should.deepEqual([ entryDate ])
```

Reply:
this suggestion is, to me, outside the scope of this test, which currently keeps it to "is there an update" aka is there a new version. The more specific the test, the less maintainable (ex: if the date format changes in the future, we would only need to change the hardcoded values at the beggining of this test). imo, this suggestion would fit for a unit test.